force-app/main/default/lwc/note/note.js

```diff
@@ -1,147 +1,229 @@
+
+    @track saveNotes = [];
+    searchDate = '';
+    noteToUpdate;
+    searchNull;
+
+    
+    connectedCallback(){
+        this.loadSavedData();
+        console.log('Work');
+    }   
+
+    handleDateChange(event) {
+        this.searchDate = event.target.value;
+    }
+    
 
 
-    //1
-    @wire(getSavedNotes)
-    wiredSavedNotes({ error, data}){
-        console.log('Wire is called');
-        if (data) {
-            this.savedData = data;
-            console.log("Result: " + data);
-        } else if (error) {
+    async loadSavedData(){
+        console.log('Loading data from server...');
+        try{
+            const result = await getSavedNotes();
+            this.saveNotes = result;
+            console.log('Saved Data: ', JSON.stringify(this.savedData));
+            console.log('Saved Data: ', JSON.stringify(result));
+        }catch (error){
             console.error('Error loading data', error);
         }
+        // console.log('Updated savedData: ', this.savedData);
     }
-    // connectedCallback(){
-
-    //     const savedData = localStorage.getItem('savedData'); //retrive from database
-    //     if (savedData) {
-    //         this.savedData = JSON.parse(savedData);
-    //     }
-    // }
-    
 
     async handleAdd(){
         const result = await MyModal.open({
             size: 'Small',
             label: 'Note',
             description: 'This is a modal popup'
         });
-        
-        // this.result = result;
-        // console.log('this.result = ' + this.result);  
-         
-    
-        if (result) {
-
-            console.log('Result 2: ' + result);
-            saveData({
-                label: result.label,
-                description: result.description
-            })
-            .then((savedNoteId)=> {
-                if(savedNoteId) {
-                    
-                    // getSavedNotes()
-                    //     .then((result)=>{
-                    //         this.savedData = result;
-                    //     })
-                    //     .catch((error) => {
-                    //         console.error('Error fetching saved data', error);
-                    //     });
-                    this.savedData = [...this.savedData,{
-                        Id: savedNoteId,
-                        Label__c: result.label,
-                        Description__c: result.description,
-                        Created_Date__c: new Date().toISOString()
-                    }];
-
-                    // result.Id = savedNote;
-                    // this.savedData = [...this.savedData, savedNote]
-                    this.dispatchEvent(
-                        new ShowToastEvent({
-                            title: 'Success',
-                            message: 'Data saved successfully',
-                            variant: 'success'
-                        })
-                    );
+        console.log('work' +  result);
+            try{
```

Comment:
You can also use here: 
`saveData({label: '', description: ''})
    .then(() => {
        //show toast
        //load data
    }).catch(error => {
       //handle error
    });
    `